hack/verify-pylint.sh

```diff
@@ -18,7 +18,7 @@ set -o nounset
 set -o pipefail
 
 export PYLINTHOME=$(mktemp -d)
-pylint="$(dirname $0)/pylint_bin"
+pylint="$(dirname $0)/pylint_bin.py"
```

Comment:
this breaks the bazel test, the pylint_bin is build and includes some libraries, since pylint imports libraries when linting.